cmd/helpers.go

```diff
@@ -299,6 +305,39 @@ func relabelEligibleTorrents(ctx context.Context, log *logrus.Entry, c client.In
 	return nil
 }
 
+func relabelTorrent(ctx context.Context, log *logrus.Entry, c client.Interface, t config.Torrent, label string, hardlink bool, noti notification.Sender, relabeledTorrents *int, errorRelabelTorrents *int, fields *[]notification.Field) {
+	if !t.APIDividerPrinted {
+		log.Info("-----")
+	}
+
+	if hardlink {
```

Comment:
Please get rid of the hardlink naming as it is not a hardlink, the code is about file overlaps since relabel has no concept of hardlinked files, yet.